src/main.ts

```
@@ -39,9 +39,20 @@ import { AllExceptionsFilter } from './common/filters/exception.filter';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+
+  // Configure raw body for webhook routes
+  app.use(
+    '/user/v1/payments/webhook/stripe',
```

ESLint

[error] Replace `'/user/v1/payments/webhook/stripe'` with `"/user/v1/payments/webhook/stripe"`

(prettier/prettier)